enterprise_subsidy/apps/api/v1/views/subsidy.py

```diff
@@ -326,12 +325,13 @@ def create(self, request, *args, **kwargs):
     )
     def destroy(self, request, *args, **kwargs):
         """
-        Delete a subsidy
+        Soft delete a subsidy by setting its is_soft_deleted field to True
 
         Endpoint Location: DELETE /api/v1/subsidies/{uuid}/
         """
-        response = super().destroy(request, kwargs['uuid'])
-        return response
+        instance = self.get_object()
+        instance.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Just curious, did you try leaving this method body alone?  I kinda figured that `super().destroy(...)` would basically do the same thing that you're explicitly doing here in this `destroy` view.